src/content/get-started/install/deepsparse-ent.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: "DeepSparse Enterprise"
+metaTitle: "DeepSparse Enterprise Installation"
+metaDescription: "Installation instructions for the DeepSparse Engine enabling performant neural network deployments"
+githubURL: "https://github.com/neuralmagic/docs/blob/main/src/content/install/deepsparse.mdx"
+index: 2000
+---
+
+# DeepSparse Enterprise Edition Installation
+
+The [DeepSparse Engine](/products/deepsparse-ent) enables GPU-class performance on CPUs, leveraging sparsity within models to reduce FLOPs and the unique cache hierarchy on CPUs to reduce memory movement.
+The engine accepts models in the open-source [ONNX format](https://onnx.ai/), which are easily created from PyTorch and TensorFlow models.
+
+Currently, DeepSparse is tested on Python 3.7-3.9, ONNX 1.5.0-1.10.1, ONNX opset version 11+ and is [manylinux compliant](https://peps.python.org/pep-0513/).
+It is limited to Linux systems running on x86 CPU architectures.
+
+The DeepSparse Engine is available in two editions:
+1. [**The Community Edition**](/products/deepsparse) is open-source and free for evaluation, research, and non-production use with our [Engine Community License](https://neuralmagic.com/legal/engine-license-agreement/).
+2. [**The Enterprise Edition**](/products/deepsparse-ent) requires a Trial License or [can be fully licensed](https://neuralmagic.com/legal/master-software-license-and-service-agreement/) for production, commercial applications.
+
+## General Install
+
+Use the following command to install with pip:
+
+```bash
+pip install deepsparse-ent
+```
+
+## Server Install
+
+The [DeepSparse Server](/use-cases/deploying-deepsparse/deepsparse-server) allows you to serve models and pipelines through an HTTP interface using the deepsparse.server CLI.
+To install, use the following extra option:
+
+```bash
+pip install deepsparse-ent[server]
+```
+
+## YOLO Install
+
+The [Ultralytics YOLOv5](/use-cases/object-detection/deploying) models require extra dependencies for deployment.
+To use YOLO models, install with the following extra option:
+
+```bash
+pip install deepsparse-ent[yolo]         # just yolo requirements
```

Comment:
Doesn't yolo auto install on usage?

Reply:
There are dependencies installed with the yolo flow. For 1.3, we'll be fully removing all auto-install functionality to make this consistent and expected from the user side.